tests/test_packages/test_connections/test_p2p_libp2p/test_public_dht.py

```diff
@@ -43,44 +42,25 @@
     PUBLIC_ID as P2P_CLIENT_CONNECTION_PUBLIC_ID,
 )
 
-from tests.conftest import (
+from tests.conftest import DEFAULT_LEDGER
+from tests.test_packages.test_connections.test_p2p_libp2p.base import (
     BaseP2PLibp2pTest,
-    DEFAULT_LEDGER,
-    DEFAULT_LEDGER_LIBP2P_NODE,
-    PUBLIC_DHT_DELEGATE_URI_1,
-    PUBLIC_DHT_DELEGATE_URI_2,
-    PUBLIC_DHT_P2P_MADDR_1,
-    PUBLIC_DHT_P2P_MADDR_2,
-    PUBLIC_DHT_P2P_PUBLIC_KEY_1,
-    PUBLIC_DHT_P2P_PUBLIC_KEY_2,
-    PUBLIC_STAGING_DHT_DELEGATE_URI_1,
-    PUBLIC_STAGING_DHT_DELEGATE_URI_2,
-    PUBLIC_STAGING_DHT_P2P_MADDR_1,
-    PUBLIC_STAGING_DHT_P2P_MADDR_2,
-    PUBLIC_STAGING_DHT_P2P_PUBLIC_KEY_1,
-    PUBLIC_STAGING_DHT_P2P_PUBLIC_KEY_2,
+    LIBP2P_LEDGER,
     _make_libp2p_client_connection,
     _make_libp2p_connection,
+    libp2p_log_on_failure_all,
+    load_client_connection_yaml_config,
+    ports,
 )
-from tests.conftest import default_ports as ports
-from tests.conftest import libp2p_log_on_failure_all
-
-
-PUBLIC_DHT_MADDRS = [PUBLIC_DHT_P2P_MADDR_1, PUBLIC_DHT_P2P_MADDR_2]
-PUBLIC_DHT_DELEGATE_URIS = [PUBLIC_DHT_DELEGATE_URI_1, PUBLIC_DHT_DELEGATE_URI_2]
-PUBLIC_DHT_PUBLIC_KEYS = [PUBLIC_DHT_P2P_PUBLIC_KEY_1, PUBLIC_DHT_P2P_PUBLIC_KEY_2]
-PUBLIC_STAGING_DHT_MADDRS = [
-    PUBLIC_STAGING_DHT_P2P_MADDR_1,
-    PUBLIC_STAGING_DHT_P2P_MADDR_2,
-]
-PUBLIC_STAGING_DHT_DELEGATE_URIS = [
-    PUBLIC_STAGING_DHT_DELEGATE_URI_1,
-    PUBLIC_STAGING_DHT_DELEGATE_URI_2,
-]
-PUBLIC_STAGING_DHT_PUBLIC_KEYS = [
-    PUBLIC_STAGING_DHT_P2P_PUBLIC_KEY_1,
-    PUBLIC_STAGING_DHT_P2P_PUBLIC_KEY_2,
-]
+
+
+genesis_nodes = load_client_connection_yaml_config()["nodes"]
```

Comment:
this information is now read from the corresponding yaml instead of being a hardcoded duplicate in tests